File: ejercicios/python/ejercicio080.py
import random

lista = []
for i in range(10):
    lista.append(random.randint(-50, 50))

positivos = 0
negativos = 0

for e in lista:
    if e >= 0:
        positivos = positivos + e
    else:
        negativos = negativos + e
    

print(lista)
print("la suma de los positivos es: " + str(positivos))
print("la suma de los negativos es: " + str(negativos))

File: ejercicios/c++/ejercicio080.cpp
#include <iostream>
#include <ctime>
#include <cstdlib>
using namespace std;

int main(){
	int ar[10];
	int cont_positivos = 0, cont_negativos = 0;
	srand(time(0));
	
	for(int i = 0; i < 10; i++){
		ar[i] = (rand() % 100) - 50;
	}
	
	for(int i = 0; i < 10; i++){
		if(ar[i] > 0)
			cont_positivos += ar[i];
		else
			cont_negativos += ar[i];
	}
	
	cout << "Elementos del arreglo: " << endl;
	for(int i=0;i<10;i++)
		cout << ar[i] << "  "; 
	
	cout << endl;
	
	cout << " los positivos suman : " << cont_positivos << endl;
	cout << " los negativos suman : " << cont_negativos << endl;
	
	return 0;
}
